apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -302,4 +360,82 @@ deserialize_overlay_test() ->
     Msg = mesos_state_overlay_pb:decode_msg(OverlayData, mesos_state_agentinfo),
     ?assertEqual(<<"10.0.0.160:5051">>, Msg#mesos_state_agentinfo.ip).
 
+vxlan_match_test() ->
+    VNI = 1024,
+    VTEPIP = <<"44.128.0.1/20">>,
+    VTEPMAC = <<"70:b3:d5:80:00:01">>,
+    VTEPMAC2 = <<"70:b3:d5:80:00:02">>,
+    VTEPNAME = <<"vtep1024">>,
+    FAKE_LINK_INFO = [
```

Comment:
that's very long data: we have a style checker that has 30-line limit for functions, but unfortunately it doesn't force the rule for `TEST` code. I'd recommend to move this data to `apps/dcos_l4lb/test/vtep_link.data` and read that file here using `file:consult/1`.